src/components/Details/VmDetails/fixtures/VmDetails.fixture.js

```diff
@@ -113,20 +116,34 @@ const vmiFixture = {
   },
 };
 
+const k8sGet = () =>
+  new Promise(resolve => {
+    resolve(fedora28);
+  });
+
+const overviewStyle = {
```

Comment:
Is there a specific reason to have a set width for the component? I think it would leave more room for the experimentation/exploration of component if we omit that. The user can always use the browser to change viewport size.

Reply:
ok, i removed additional styling from fixture